src/shared/utils/project-utils.ts

```diff
@@ -11,11 +11,15 @@ import { prefixPlaygroundAssetsURL, extractPageMetadata } from './uidl-utils'
 import { slugify, sanitizeVariableName } from './string-utils'
 import { FILE_EXTENSIONS } from '../constants'
 
-export const createHtmlIndex = (
-  uidl: ProjectUIDL,
-  assetsPrefix: string,
+interface HtmlIndexFileParams {
+  uidl: ProjectUIDL
```

Comment:
just a small thing, would it make more sense to have two params to this function? - `projectUIDL` and `options`. it would make the API clearer, because the UIDL is mandatory, all the rest are optional fields that can be sent or not